src/Symfony/Component/Workflow/Registry.php

```diff
@@ -23,19 +24,26 @@ class Registry
 
     /**
      * @param Workflow $workflow
-     * @param string   $className
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $className)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
K, will do :)

Should I revert changes in Extension back to inject class name string only instead of wrapping it into the ClassInstanceSupportStrategy? Or keep them as that might be the target for 4.0 anyways?